randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -844,8 +846,11 @@ Extra - room_id: [26]
   > Door to Security Access (Upper)
       Space Jump
   > Door to Crumble City
-      # TODO: Look into requirements further and possibly add SWJ trick. The requirement of space jump makes the advanced tricks in crumble city invalid in non-entrance rando.
-      Screw Attack and Space Jump
+      All of the following:
+          Screw Attack
+          Any of the following:
+              Space Jump or Can Single Walljump
+              Level 1 Locks and Speed Booster and Shinespark Tricks (Hypermode) and Disabled Door Lock Rando and Disabled Entrance Rando
```

Comment:
A) you also need L0
B) you can do this without l1 locks, which is even harder. Could probably have the l1 behind expert but maybe knowledge? IDK how known it is that you can jump out of it.

Reply:
I'm apprehensive to add a knowledge requirement here just because its hypermode/expert tbh.
If we did, whos to say that we don't add a knowledge requirement to literally every shinespark in the game?
Also chances are if you are setting shinepark trick to expert, you're going to be experimenting with it a lot